iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/SendMessagesIT.java

```diff
@@ -5,16 +5,36 @@
 
 package com.microsoft.azure.sdk.iot.android;
 
```

Comment:
Make sure all the tests here matches what exist on jvm. If there are additional tests please move them here as well.